src/RegExLib.Web/Endpoints/Expressions/Create.cs

```diff
@@ -0,0 +1,35 @@
+﻿using Ardalis.ApiEndpoints;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Threading.Tasks;
+using RegExLib.SharedKernel.Interfaces;
+using RegExLib.Web.ApiModels;
+
+namespace RegExLib.Web.Endpoints.Expressions
+{
+  public class Create : BaseAsyncEndpoint<CreateExpressionCommand, CreateExpressionResult>
+  {
+    private readonly IRepository _repository;
+
+    public Create(IRepository repository)
+    {
+      _repository = repository;
+    }
+
+    [HttpPost("/api/expressions")]
+    [SwaggerOperation(
+      Summary = "Creates a new Expression",
+      Description = "Creates a new Expression",
+      OperationId = "Expression.Create",
+      Tags = new[] { "ExpressionEndpoint" })
+    ]
+    public override async Task<ActionResult<CreateExpressionResult>> HandleAsync([FromBody] CreateExpressionCommand request)
+    {
+      var expression = ExpressionDTO.ToExpression(request.ExpressionDto);
+      expression = await _repository.AddAsync(expression);
```

Comment:
Reassigning `expression` makes the code harder to follow. I'd prefer seeing something like `var createdExpression = ...`.

Reply:
Done in [commit effacc2](https://github.com/ardalis/RegExLib.com/pull/44/commits/effacc281c598ff5eb9a31e5d099fa6a41a1ced2)